src/handler.py

```diff
@@ -135,7 +154,266 @@ def restore_db_cluster(event, context):
+        if my_env_variables.get("AQTS_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["AQTS_SCHEMA_OWNER_PASSWORD"] = db_password
+        elif my_env_variables.get("TRANSFORM_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["TRANSFORM_SCHEMA_OWNER_PASSWORD"] = db_password
+        if my_env_variables.get("AQTS_DATABASE_ADDRESS") is not None:
+            my_env_variables["AQTS_DATABASE_ADDRESS"] = db_address
+        elif my_env_variables.get("TRANSFORM_DATABASE_ADDRESS") is not None:
+            my_env_variables["TRANSFORM_DATABASE_ADDRESS"] = db_address
+        if my_env_variables.get("DB_PASSWORD") is not None:
+            my_env_variables["DB_PASSWORD"] = db_password
+        if my_env_variables.get("DB_HOST") is not None:
+            my_env_variables["DB_HOST"] = db_address
+        logger.info(f"AFTER function {lambda_function} my_env_variables= {my_env_variables}")
+
+        # lambda_client.update_function_configuration(
+        #     FunctionName=lambda_function,
+        #     Environment={
+        #         'Variables': my_env_variables
+        #     }
+        # )
+
+
+def restore_secrets(event, context):
+
+    original = secrets_client.get_secret_value(
+         SecretId=NWCAPTURE_TEST
+    )
+    secret_string = json.loads(original['SecretString'])
+    db_password = str(secret_string['SCHEMA_OWNER_PASSWORD'])
+    db_address = str(secret_string['DATABASE_ADDRESS'])
+    logger.info(f"db_address {db_address} db_password {db_password}")
+
+    for lambda_function in LAMBDA_FUNCTIONS:
+
+        response = lambda_client.get_function_configuration(
+            FunctionName=lambda_function
+        )
+        my_env_variables = response['Environment']['Variables']
+        logger.info(f"BEFORE function {lambda_function} my_env_variables= {my_env_variables}")
+        if my_env_variables.get("AQTS_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["AQTS_SCHEMA_OWNER_PASSWORD"] = db_password
+        elif my_env_variables.get("TRANSFORM_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["TRANSFORM_SCHEMA_OWNER_PASSWORD"] = db_password
+        if my_env_variables.get("AQTS_DATABASE_ADDRESS") is not None:
+            my_env_variables["AQTS_DATABASE_ADDRESS"] = db_address
+        elif my_env_variables.get("TRANSFORM_DATABASE_ADDRESS") is not None:
+            my_env_variables["TRANSFORM_DATABASE_ADDRESS"] = db_address
+        if my_env_variables.get("DB_PASSWORD") is not None:
+            my_env_variables["DB_PASSWORD"] = db_password
+        if my_env_variables.get("DB_HOST") is not None:
+            my_env_variables["DB_HOST"] = db_address
+
+        logger.info(f"AFTER function {lambda_function} my_env_variables= {my_env_variables}")
+
+        # lambda_client.update_function_configuration(
+        #     FunctionName=lambda_function,
+        #     Environment={
+        #         'Variables': my_env_variables
+        #     }
+        # )
```

Comment:
Is your intention to uncomment this before merging?

Reply:
yes, thanks!